services/kubecost/0.37.2/defaults/cm.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: kubecost-0.37.1-d2iq-defaults
+  name: kubecost-0.37.2-d2iq-defaults
```

Comment:
and here as well

Reply:
I thought this was required (reporting the comment), so I did it everywhere. https://github.com/mesosphere/kommander-applications/pull/1785#discussion_r1434016668

Glad to rollback if not needed.